backend/app.js

```diff
@@ -13,20 +11,28 @@ app.use(bodyParser.json());
 // not used here, but you can parse urlencoded data too
 app.use(bodyParser.urlencoded({ extended: false }));
 
-var corsOption = {
-  origin: true,
-  methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
-  credentials: true,
-  exposedHeaders: ['x-auth-token']
-};
-app.use(cors(corsOption));

```

Comment:
Should not have been deleted, refer to master branch to see final changes. Don't remove code from that unless you meant to modify it